lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/OpenSearchAppender.java

```diff
@@ -0,0 +1,17 @@
+package org.enso.logging.service.logback;
+
+import java.util.ServiceLoader;
+
+public abstract class OpenSearchAppender extends RemoteAppender {
+
+  static RemoteAppender load() {
+    var loader = ServiceLoader.load(RemoteAppender.class, RemoteAppender.class.getClassLoader());
+    var appender =
+        loader.stream()
+            .filter(l -> l.type().getName().contains("OpenSearch"))
```

Comment:
Searching for a class name is ugly. Some might say this is even more _loosely coupled_, but it is ugly.